lang/en/block_equella_tasks.php

```diff
@@ -19,5 +19,4 @@
 $string['incompatible'] = 'This version of openEQUELLA is incompatible with this block';
 $string['error'] = 'There has been an error connecting to openEQUELLA.  Will try again in 5 minutes.';
 $string['notasks'] = 'You currently have no tasks';
-
-?>

```

Comment:
I'm surprised this was happy. You still have the opening `<?php` but then you've chopped this one off. Could be good to return it. :thinking: